pom.xml

```diff
@@ -115,6 +115,12 @@
 			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
 			<version>2.5.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.struts</groupId>
```

Comment:
why do you need this dependency?

Reply:
This dependency was added for using @FieldMatch in UserRegistrationRequestDto and not writing custom annotation. But validating two fields (password & repeatPassword) for a match still doesn't work